client/me/security-2fa-disable/index.jsx

```diff
@@ -141,17 +143,17 @@ class Security2faDisable extends Component {
 					<p>
 						{ translate(
 							'This will also disable your Application Passwords, ' +
-								'though they will be available again if you choose to re-enable ' +
-								'Two-Step Authentication in the future. If you do choose to ' +
-								're-enable it, please note that you will need to generate ' +
-								'new backup codes.'
+								'though you can access them again if you ever re-enable ' +
+								'Two-Step Authentication. If you decide to re-enable ' +
+								"Two-Step Authentication, keep in mind you'll need to " +
+								'generate new backup codes.'
 						) }
 					</p>
 					<p>
 						{ translate(
-							'To verify that you wish to disable Two-Step ' +
-								'Authentication, please enter the verification code from your ' +
-								'device or a backup code and click "Disable Two-Step."'
+							'To verify that you wish to disable Two-Step Authentication, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'To verify that you wish to disable Two-Step Authentication, please enter the verification code from your device or a backup code and click "Disable Two-Step."' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/me-security-2fa-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@benhuberman how do we typically do with these? The difference between the old one vs new one is `please` and one `,`.

Old:
> To verify that you wish to disable Two-Step Authentication, please enter the verification code from your device or a backup code and click "Disable Two-Step."

New:
> To verify that you wish to disable Two-Step Authentication, enter the verification code from your device or a backup code, and click "Disable Two-Step."